src/edge/edge.svg

```diff
@@ -1 +1,39 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 650 650"><path d="M586 307c0-166-116-280-270-282C161 27 53 138 33 293c56-92 167-154 265-154 80 0 123 43 123 128H207c4-36 20-73 42-95-86 35-146 113-146 218 0 130 100 235 246 235 74 0 134-16 186-43V455c-47 28-103 45-165 45-93 0-160-45-160-127h376z" fill="#0078d7"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 256 256">
```

Comment:
This should be run through [SVGOMG](https://jakearchibald.github.io/svgomg/).

Reply:
It seems it is run through SVGOMG, but maybe not with all possible options enabled :) Especially prettify is on :)

Reply:
Thank you and done. I have taken the original SVG and run it through SVGOMG with every feature enabled, except “remove xmlns", and with precision set to `3`, at which I saw no noticeable change when zoomed in.